numerical_methods/rungekutta.cpp

```diff
@@ -0,0 +1,115 @@
+/**
+ *  \addtogroup Runge Kutta 4th Method for solving Differential equation
+ *  @{
+ *  \file
+ *  \brief Runge Kutta Method implementation
+ *
+ *  \author [Rudra Prasad Das](http://github.com/rudra697)
+ *
+ *  \details
+ * It solves the unknown value of y 
+ * for a given value of x
+ * only first oreder differential equations
+ *can be solved
+ *  \example
+ *  it solves dy/dx=(x-y)/2 at a 
+ * given x for given initial 
+ * conditions
+ * **/
+#include<iostream> 
+/**
+ * The change() function is used 
+ * to return the updated iterative value corresponding 
+ * to the given function
+ * @param x is the value corresponding to the x coordinate
+ * @param y is the value corresponding to the y coordinate 
+ * @return the computed function value at that call
+ * **/
+double change(double x, double y) 
+{ 
+	double val=((x - y)/2.0); 
+	return val;
+} 
+/**
+ * 
+ *   The rungeKutta finds value of y for 
+ * a given x using step size h 
+ * and initial value initial x at initial y. 
+ * @param init_x is the value of initial x and is updated after each call
+ * @param init_y is the value of initial x and is updated after each call
+ * @param x is the given value of x
+ * @param h is the step value 
+ * @param n is the count of iterations
+ * @param k1 is the first variable of runge kutta method
+ * @param k2 is the second variable of runge kutta method
+ * @param k3 is the third variable of runge kutta method
+ * @param k4 is the fourth variable of runge kutta method
+ * @param y is the final value of y after solving the differential equation
```

Comment:
These are not parameters, these are local variables.
```suggestion
```

Reply:
These lines have been removed and commented briefly inside the function